src/Pages/Home.tsx

```diff
@@ -79,6 +80,29 @@ const departments: Department[] = [
   { name: "Instrumentos Musicales", icon: "ph:guitar-light", discount: "35%" },
 ];
 
+const homeCardData: Array<homeCardProps> = [
```

Comment:
Para `type` e `interface` se usa pasal case

```suggestion
const homeCardData: Array<HomeCardProps> = [
```